<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>-10</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>请选择左手揲馀的蓍草数量（共十八次）</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>（三变成一爻，十八变成一卦）</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>130</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>占卜结果</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>160</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>主卦（爻）辞：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>240</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>副卦（爻）辞：</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="guaci1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>180</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="guaci2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="chongzhi">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>320</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>195</width>
      <height>65</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>291</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>当前爻象</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>本卦卦形</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>之卦卦形</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTextBrowser" name="yao6"/>
     </item>
     <item row="1" column="1">
      <widget class="QTextBrowser" name="ben6"/>
     </item>
     <item row="1" column="2">
      <widget class="QTextBrowser" name="bian6"/>
     </item>
     <item row="2" column="0">
      <widget class="QTextBrowser" name="yao5"/>
     </item>
     <item row="2" column="1">
      <widget class="QTextBrowser" name="ben5"/>
     </item>
     <item row="2" column="2">
      <widget class="QTextBrowser" name="bian5"/>
     </item>
     <item row="3" column="0">
      <widget class="QTextBrowser" name="yao4"/>
     </item>
     <item row="3" column="1">
      <widget class="QTextBrowser" name="ben4"/>
     </item>
     <item row="3" column="2">
      <widget class="QTextBrowser" name="bian4"/>
     </item>
     <item row="4" column="0">
      <widget class="QTextBrowser" name="yao3"/>
     </item>
     <item row="4" column="1">
      <widget class="QTextBrowser" name="ben3"/>
     </item>
     <item row="4" column="2">
      <widget class="QTextBrowser" name="bian3"/>
     </item>
     <item row="5" column="0">
      <widget class="QTextBrowser" name="yao2"/>
     </item>
     <item row="5" column="1">
      <widget class="QTextBrowser" name="ben2"/>
     </item>
     <item row="5" column="2">
      <widget class="QTextBrowser" name="bian2"/>
     </item>
     <item row="6" column="0">
      <widget class="QTextBrowser" name="yao1"/>
     </item>
     <item row="6" column="1">
      <widget class="QTextBrowser" name="ben1"/>
     </item>
     <item row="6" column="2">
      <widget class="QTextBrowser" name="bian1"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="text">
        <string>变数：</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="bianshu"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>684</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>蓍草占卜</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
